app/Filament/Resources/EmulatorResource.php

```diff
@@ -0,0 +1,234 @@
+                                        ->label('Original name')
+                                        ->helperText('Original name of emulator.'),
+                                    Infolists\Components\TextEntry::make('description')
+                                        ->label('Description')
+                                        ->helperText('Additional text to display on the download page.'),
+                                    Infolists\Components\TextEntry::make('link')
+                                        ->label('Documentation link')
+                                        ->helperText('Link to emulator documentation.'),
+                                ]),
+                        ]),
+                    Infolists\Components\Section::make([
+                        Infolists\Components\TextEntry::make('id')
+                            ->label('ID'),
+                        Infolists\Components\TextEntry::make('Created')
+                            ->label('Created at')
+                            ->dateTime()
+                            ->hidden(fn ($state) => !$state),
+                        Infolists\Components\TextEntry::make('Updated')
+                            ->label('Updated at')
+                            ->dateTime()
+                            ->hidden(fn ($state) => !$state),
+                        Infolists\Components\IconEntry::make('active')
+                            ->boolean(),
+                    ])->grow(false),
+                ])->from('md'),
+            ]);
+    }
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->columns(1)
+            ->schema([
+                Forms\Components\Split::make([
+                    Forms\Components\Section::make()
+                        ->columns(2)
+                        ->schema([
+                            Forms\Components\TextInput::make('handle')
+                                ->label('Name')
+                                ->required()
+                                ->maxLength(255)
+                                ->helperText('Name of emulator'),
+                            Forms\Components\TextInput::make('name')
+                                ->label('Original name')
+                                ->required()
+                                ->maxLength(255)
+                                ->helperText('Original name of emulator.'),
+                            Forms\Components\Textarea::make('description')
+                                ->label('Description')
+                                ->rows(5)
+                                ->helperText('Additional text to display on the download page.'),
+                            Forms\Components\TextInput::make('link')
+                                ->label('Documentation link')
+                                ->helperText('Link to emulator documentation.'),
```

Comment:
Would a `->url()` validator be appropriate for this field?

Reply:
yes.